app/src/test/java/com/asfoundation/wallet/repository/ApproveServiceTest.java

```diff
@@ -35,6 +36,7 @@
   @Mock TrackTransactionService trackTransactionService;
   @Mock TransactionSender transactionSender;
   @Mock TransactionValidator transactionValidator;
+  @Mock Gson gson;
```

Comment:
We could use converter from retrofit, this makes it agnostic with the json parser that we use (especially since Gson seems abandoned and Moshi is the "replacement")